molecule/testinfra/staging/vars/staging.yml

```diff
@@ -12,8 +12,8 @@ wanted_apache_headers:
   - 'Header unset Etag'
 
 securedrop_venv: /opt/venvs/securedrop-app-code
-securedrop_venv_bin: "/opt/venvs/securedrop-app-code/bin"
-securedrop_venv_site_packages: "/opt/venvs/securedrop-app-code/lib/python3.5/site-packages"
+securedrop_venv_bin: "{{ securedrop_venv }}/bin"
+securedrop_venv_site_packages: "{{ securedrop_venv }}/lib/python3.5/site-packages"
```

Comment:
These two lines are breaking the CI (`testinfra` tests)

Reply:
Because this YAML file is loaded into Python as a dict, so no `jinja` style var substitution there.

Reply:
reverted to plain strings.